src/setup/scss/_reset.scss

```diff
@@ -1,4 +1,4 @@
-@import '~reset-css/reset.css';
+@import '~the-new-css-reset/css/reset.css';
 
```

Comment:
todo: please drop this file and move the `the-new-css-reset` directly to `setup/_scss.scss`. I don't think we will need additional resets beside the once provided. For basic styles we should use `globals.scss` instead.

thought: maybe we will have to think about a new structure anyway. e.g.
- using `setup/style.scss` instead of `setup/_scss.scss`
- splitting `global.scss` into `basic.scss` (styling native elements) and `global.scss` (providing global classes like `.wysiwyg` or `.invisible`.